frontend/layouts/default.vue

```diff
@@ -115,6 +118,13 @@ export default Vue.extend({
             return this.$nuxt?.context?.isDev;
         },
     },
+    watch: {
+        getMakerNetwork(newValue) {
+            if (newValue) {
+                this.$store.dispatch('network/setup');
+            }
+        },
```

Comment:
Would this also be called on the page load (after persistent values are inserted into the store)? – This will result in doubling the amount of requests.

Should we call the same thing `network/setup` initially? Currently [something else is called here](https://github.com/sidestream-tech/unified-auctions-ui/blob/8d976c1c74618c3179059c311d232d2dcf2ee949/frontend/plugins/vuex-persist.client.js#L18-L21) (because of the problems with persistent storage which get initialised sometimes before, sometimes after the layout mount)